pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -565,6 +573,18 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 			ps.CheckMaxTimeTopic(agentName, "publishTimer", start,
 				warningTime, errorTime)
 			start = time.Now()
+			capabilities, err := hyper.GetCapabilities()
```

Comment:
Nit, but why publish this periodically? Do we expect it to change on a running system?

Reply:
That was exactly the same question I was asking ;-)

Reply:
I left this in case there was an error in getting or sending the capability data.
Made changes, now the check of getting and sending is explicit.

Reply:
Looks fine in latest diffs.